MobileFilter/sections/specific_web.txt

```diff
@@ -893,6 +893,7 @@ profile.ru##div[data-typeitem="ad"]
 rbc.ua##.adsens-wrap
 app.famitsu.com##.ad-agency-area
 app.famitsu.com###gam_sp_header_banner_1
+app.famitsu.com##.gam_sp_post_content_after_2
```

Comment:
Looks typo, probably meant `###gam_sp_post_content_after_2`

Reply:
sorry. I will fix it right now.